### PR TITLE
[DC-706] Modify parameters of cleaning rule from DC-584

### PR DESCRIPTION
**Pull request by nishanthpp93**
* [DC-706] Separate out queries for missing basics and missing ehr data
* [DC-706] Consider all pid tables for deletion
* [DC-706] Sandbox person rows before deleting
* [DC-706] Use dict and add back basics constant

[DC-706]: https://precisionmedicineinitiative.atlassian.net/browse/DC-706
[DC-706]: https://precisionmedicineinitiative.atlassian.net/browse/DC-706
[DC-706]: https://precisionmedicineinitiative.atlassian.net/browse/DC-706
[DC-706]: https://precisionmedicineinitiative.atlassian.net/browse/DC-706

### CODE REVIEW

**Review by jp3477 — CHANGES_REQUESTED** [review]
I added a note about converting the cleaning rules to the base class implementation.
Additionally, one of the integration tests, `drop_participants_without_ppi_or_ehr_test.py` is failing and needs to be updated. Other than that, everything looks good.

File: data_steward/cdr_cleaner/cleaning_rules/drop_participants_without_ppi_or_ehr.py [hunk 1]
```diff
@@ -9,88 +9,121 @@
...
-  basics.person_id IS NOT NULL
-  {% for table in mapped_clinical_data_configs.keys() %}
-    OR {{ table }}_ehr.person_id IS NOT NULL
-  {% endfor %}
+)
 """)
 
 
-def get_queries(project=None, dataset=None, sandbox_dataset_id=None):
+def get_queries(project_id=None, dataset_id=None, sandbox_dataset_id=None):
```
[the remark this answers is not in the document]
Comment (nishanthpp93) [reply data_steward/cdr_cleaner/cleaning_rules/drop_participants_without_ppi_or_ehr.py:76]: Thanks, I missed this. I'll make that change

**Review by jp3477 — APPROVED** [review]
This looks good! Thanks for the changes.

**Comment by nishanthpp93** [comment]
There seems to be an unrelated test that's failing. Not sure why.

**Pull request merged**